frontend/src/components/dashboard/aliases/Alias.tsx

```diff
@@ -58,7 +58,7 @@ export const Alias = (props: Props) => {
     expandButtonRef
   ).buttonProps;
 
-  const address = getFullAddress(props.alias, props.profile);
+  const address = getFullAddress(props.alias);
```

Comment:
Other reviewers - the change to ``getFullAddress`` is in [frontend/src/hooks/api/aliases.ts](https://github.com/mozilla/fx-private-relay/pull/1677/files#diff-4c057af7d1efdc54dd80dcba1408bf6a03464cb4b800f10d13d7a9943d7c0092).